src/js/node/tls.js

```diff
@@ -30,15 +30,19 @@ function parseCertString() {
   throwNotImplemented("Not implemented");
 }
 
+const rejectUnauthorizedDefault =
+  process.env.NODE_TLS_REJECT_UNAUTHORIZED !== "0" && process.env.NODE_TLS_REJECT_UNAUTHORIZED !== "false";
 function isValidTLSArray(obj) {
   if (typeof obj === "string" || isTypedArray(obj) || obj instanceof ArrayBuffer || obj instanceof Blob) return true;
   if (Array.isArray(obj)) {
     for (var i = 0; i < obj.length; i++) {
-      if (typeof obj !== "string" && !isTypedArray(obj) && !(obj instanceof ArrayBuffer) && !(obj instanceof Blob))
+      const item = obj[i];
```

Comment:
where is the test that makes sure we don't make this mistake again?

Reply:
some here https://github.com/oven-sh/bun/blob/05bc3b68bb5b3832674cd7b626da94a86d54d7bc/test/js/node/tls/node-tls-cert.test.ts#L164-L215

We already have tests that uses Buffer or string too, but I can provide additional tests.